app/services/client.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+
+class client:
```

Comment:
class name should be in PascalCase, for example: WordsClient
and specify a more meaningful name, so it would be more clear what this client is intended to do